### ISSUE TITLE
Lightsticks fall through the floor

### ISSUE THREAD

**Issue by MattyDienhoff** [eezstreet/SWATEliteForce#275]
Now that they're thrown like a grenade rather than dropped in place, my lightsticks often fall through the floor unless I throw them with very little force and/or a high arc so that they land softly. The physics could use a little tweaking, if possible. Worth noting: I was on the second floor of a two story building when it happened.

**Comment by eezstreet** [comment]
Unfortunately I'm not sure what causes this. SWAT 4's Havok physics have always been wonky.

**Comment by kevinfoley** [comment]
They've always had an issue with falling through the floor; it's not just because you can throw them.

**Comment by MattyDienhoff** [comment]
Fair enough, I suppose it just doesn't happen as often when they were simply dropped, so I prefer it that way.

**Comment by dedmen** [comment]
Maybe too low mass? or too small collision mesh? This is basically making Lightsticks useless. 8/10 just fall through the ground. And the two others properly land on the ground.. But one of them instantly despawns after half a second..

**Comment by eezstreet** [comment]
Have you tried playing v7? The lightsticks there are significantly different.
________________________________

**Comment by dedmen** [comment]
Negative I'm on current stable v6.4.

**Comment by dedmen** [comment]
Why do you close this? Are you sure this is fixed?

**Comment by sandman332** [comment]
It is pretty much fixed in v7.  Try it and you will see.  It only happens very rarely now.  Which is why the thread is closed.  You can also read the v7 changelog as it states it is fixed.

**Comment by dedmen** [comment]
Just tried in v7 alpha 2. Not fixed.

**Comment by dedmen** [comment]
Also trying out the drop lightstick hotkey.. First time worked "fine" although the stick bugged through the ground and disappeared. After that I didn't switch automatically back to main weapon like I expected. And if I did manually and tried to drop a stick again.. this happened.
https://www.twitch.tv/videos/274323550
Also at the end I'm standing there with empty hand and not switching to primary.
